src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs

```diff
@@ -159,7 +159,7 @@ public async Task LearnMoreButtonClicked_ShowInBrowserCalled()
             notification.Actions.LastOrDefault().CommandText.Should().Be(MigrationStrings.MigrationPrompt_LearnMoreButton);
             notification.Actions.LastOrDefault().Action(null);
 
-            browserService.Verify(x => x.Navigate(MigrationStrings.LearnMoreUrl), Times.Once);
+            browserService.Verify(x => x.Navigate(MigrationStrings.Url_LearnMoreUrl), Times.Once);
```

Comment:
I've added a second link to the wizard, so I changed the naming so both resources start with "URL_" so they are next to each other in the resource file.